internal/asyncapi/command_import.go

```diff
@@ -405,29 +403,28 @@ func registerSchema(details *accountDetails, topicName string, components Compon
 		}
 
 		req := srsdk.RegisterSchemaRequest{
-			Schema:     string(jsonSchema),
-			SchemaType: resolveSchemaType(components.Messages[strcase.ToCamel(topicName)+"Message"].ContentType),
+			Schema:     srsdk.PtrString(string(jsonSchema)),
+			SchemaType: srsdk.PtrString(resolveSchemaType(components.Messages[strcase.ToCamel(topicName)+"Message"].ContentType)),
 		}
-		opts := &srsdk.RegisterOpts{Normalize: optional.NewBool(false)}
-		id, err := details.srClient.Register(subject, req, opts)
+		id, err := details.srClient.Register(subject, req, false)
```

Comment:
Seems like we should change the name of "id". Looks like it returns an object that contains an ID.

Reply:
you're right. Renamed it to `resp`